id_infrastructure/firestore_uuid_table.py

```diff
@@ -32,6 +32,9 @@ def update_data_to_uuid_mappings(self, updated_mappings):
         :param updated_mappings: Mappings from data to uuids
         :type updated_mappings: dict of str -> str
         """
+
+        assert "Not safe to use as not yet protected against readtimeouts"
```

Comment:
(This was only used for migrating from the old json tables to firestore, so fine not to have this implemented. We shouldn't need to this anymore anyway)